mvp.feature

```diff
@@ -151,20 +151,20 @@ Feature: User with no Author identity in the Authority Register for Personas see
     Then they see information for Volume
     And they see information for Issue
     And they see Page Number information or an Article Number
-    # it is implicit that these fields are editable
```

Comment:
Til Scenario: `User verifies information for Journal for a Publication based on a Link`: 
Hva er `eISSN`? Vi har onlineISSN og printISSN.
Finnes ikke publishers for Journals. Hva er publisher da?